submissions/yhorodechna/side-menu-dom/js/script.js

```diff
@@ -0,0 +1,107 @@
+let _navEl;
```

Comment:
what is the reason for using underscore here?

Reply:
I 'defined' the variable as private.I don't use it in another place. That's why I used underscore. Now I changed it.